src/Grids/Grids.jl

```diff
@@ -113,6 +113,8 @@ function get_boundary_types(grid::Grid{T, N, S}) where {T, N, S}
    return get_boundary_types.(grid.axes)
 end
 
+TicksTuple(g::Grid{T, 3, Cartesian}) where {T} = (x = g.axes[1].ticks, y = g.axes[2].ticks, z = g.axes[3].ticks)
+TicksTuple(g::Grid{T, 3, Cylindrical}) where {T} = (r = g.axes[1].ticks, φ = g.axes[2].ticks, z = g.axes[3].ticks)

```

Comment:
Is this still needed if we paint objects directly to the `Grid` instead of sampling?

Reply:
Right, I think we do not need it anymore.

Reply:
Then please remove it.

Reply:
I am not yet sure. That's why I would keep it in for now.